packages/core/modules/index.d.ts

```diff
@@ -547,6 +547,24 @@ interface ConfigUtils {
   cleanJSX(jsx: any): Object;
   applyJsonLogic(logic: any, data?: any): any;
 }
+interface DefaultUtils {
+  getDefaultField(config: Config, canGetFirst: boolean, parentRuleGroupPath: IdPath | null): Field;
```

Comment:
`canGetFirst` and `parentRuleGroupPath` are optional patams. Please use `?` (like `canGetFirst?: boolean`)

Same for other utils